Source/Relativity.Test.Helpers/TestHelpersKepler/TestHelpersKepler.Interfaces/TestHelpersModule/v1/Models/FieldArtifactIdRequestModel.cs

```diff
@@ -6,9 +6,8 @@
 
 namespace TestHelpersKepler.Interfaces.TestHelpersModule.v1.Models
 {
-	public class FieldArtifactIdRequestModel
+	public class FieldArtifactIdRequestModel : RequestModel
```

Comment:
any base class i prefer to name it as BaseRequestModel so when reading code its easier to understand that its a base class.